tests/conftest.py

```diff
@@ -0,0 +1,19 @@
+import sys
+from collections.abc import Iterator
+
+import pytest
+from litestar import Litestar
+from litestar.testing import TestClient
+
+# FIXME: this is a hack, should be resolved by a better code/package structure.
+sys.path.append(".")
```

Comment:
I'd be up to set up the better code/package structure in this PR, what do you think?